utils/oomparser/oomparser.go

```diff
@@ -38,6 +38,9 @@ var firstLineRegexp *regexp.Regexp = regexp.MustCompile(
 // struct to hold file from which we obtain OomInstances
 type OomParser struct {
 	systemFile string
+
+	// Inotify event watcher.
+	watcher *inotify.Watcher
```

Comment:
Do you mean there might be a backlog of IN_MODIFY notices to the watcher if there are many changes to the file all at once?